pkg/api/controller.go

```diff
@@ -817,12 +848,26 @@ func (c *Controller) CreateUser(w http.ResponseWriter, r *http.Request, body Cre
 		FriendlyName: nil,
 		Source:       "internal",
 	}
+	invite := swag.BoolValue(body.InviteUser)
+	if invite {
+		u.Username = normalizedEmail
+		u.Email = &body.Id
+	}
 	ctx := r.Context()
 	c.LogAction(ctx, "create_user")
 	_, err := c.Auth.CreateUser(ctx, u)
+
 	if handleAPIError(w, err) {
 		return
```

Comment:
Other calls to `handleAPIError` log an error, this one does not.  Would prefer consistency here.

Reply:
I haven't written this code or modified it, but I think the reason here is that two lines before we` c.LogAction(ctx, "create_user")`. If you feel we should log more explicitly, I can definitely add that

Reply:
Yes, now that there are multiple ways to fail it seems strange to log all the new ways but not the old one.

Reply:
Done